lib/time.js

```diff
@@ -349,6 +415,219 @@ function CronTime(luxon) {
 			return date;
 		},
 
+		/**
+		 * Search backwards in time 1 minute at a time, to detect a DST forward jump.
+		 * When the jump is found, the range of the jump is investigated to check for acceptable cron times.
+		 * A pair is returned, whose first is a boolean representing if an acceptable time was found inside the jump,
+		 * and whose second is a DateTime representing the first millisecond after the jump.
+		 *
+		 * The input date is expected to be decently close to a DST jump.
+		 * Up to a day in the past is checked before an error is thrown.
+		 * @param date
+		 */
+		_findPreviousDSTJump: function (date) {
+			/** @type number */
+			let expectedMinute, expectedHour, actualMinute, actualHour;
+			/** @type DateTime */
+			let maybeJumpingPoint;
+
+			// representing one day of backwards checking. If this is hit, the input must be wrong.
+			const iterationLimit = 60 * 24;
+			let iteration = 0;
+			do {
+				if (iteration++ > iterationLimit) {
+					throw new Error(
+						`ERROR: This DST checking related function assumes the input DateTime (${date.toISO()}) is within 24 hours of a DST jump.`
+					);
+				}
+
+				expectedMinute = date.minute - 1;
+				expectedHour = date.hour - 1;
+
+				if (expectedMinute < 0) expectedMinute += 60;
+				if (expectedHour < 0) expectedHour += 60;
+
+				maybeJumpingPoint = date.minus({ minute: 1 });
+
+				actualMinute = maybeJumpingPoint.minute;
+				actualHour = maybeJumpingPoint.hour;
+			} while (expectedMinute === actualMinute && expectedHour === actualHour);
+
+			const afterJumpingPoint = maybeJumpingPoint
+				.plus({ minute: 1 }) // back to the first minute _after_ the jump
+				.set({ seconds: 0, millisecond: 0 }); // and set seconds and MS to zero.
+			const beforeJumpingPoint = afterJumpingPoint.minus({ second: 1 });
```

Comment:
it's a little tricky to follow what's going on here. I know you're searching for the time inside the jump here but why specifically one minute (with the seconds and milliseconds at 0) and one second before (not caring about milliseconds this time)? might be worth some code comments

Reply:
Comments were added! I agree.

The reason why the 'after' point got this extra precision is because it may be used for scheduling a CronJob. 
The `_checkTimeInSkippedRange` func itself only needs parameters accurate down to minutes, which is why the `before` did not need the same treatment.

(Roughly this statement was also my code comment)